packages/terra-form/src/Textarea.jsx

```diff
@@ -1,18 +1,33 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
-
 import 'terra-base/lib/baseStyles';
+
+import Utilities from './_Utilities';
 import styles from './Textarea.scss';
 
 const cx = classNames.bind(styles);
 
+const TEXTAREA_ROW_SIZES = {
+  small: 2,
+  medium: 5,
+  large: 10,
```

Comment:
should yo add the 'full' size here as well?  are referencing this hash below by the size

Reply:
Not here. For this size, full is not setting the values of the rows attribute, and is instead just making the width and height 100% through css.